source/2019-03-30-the-ember-times-issue-91.md

```diff
@@ -17,8 +17,11 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Building a Hangman Game with Ember Octane](https://www.twitch.tv/videos/400723351) 🎬
+
+Mike North, together with his wife Lisa Huang did a [life stream](https://www.twitch.tv/videos/400723351) on how to build a simple hangman game. They use Ember Octane and the Glimmer components that come with it. 
```

Comment:
I think we usually do names as Github handles like this:

```suggestion
[@mike-north](https://github.com/mike-north), together with his wife Lisa Huang did a [life stream](https://www.twitch.tv/videos/400723351) on how to build a simple hangman game. They use Ember Octane and the Glimmer components that come with it. 
```

Reply:
It will be great to link to both of our github! Thanks :)

Reply:
I am :) But this just has not updated!